en/advanced_config/static_pressure_buildup.md

```diff
@@ -4,7 +4,7 @@ Air flowing over an enclosed vehicle can cause the _static pressure_ to change w
 Depending on the location of holes/leaks in the hull, you can end up with under or overpressure (similar to a wing).
 
 The change in pressure can affect barometer measurements, leading to an inaccurate altitude estimate.
-This might manifest as the vehicle losing altitude when it stops moving in [Altitude](../flight_modes_mc/altitude.md), [Position](../flight_modes_mc/position.md) or [Mission](../flight_modes/mission.md) modes (when the vehicle stops moving the static pressure drops, the sensor reports a higher altitude, and the vehicle compensates by descending).
+This might manifest as the vehicle losing altitude when it stops moving in [Altitude](../flight_modes_mc/altitude.md), [Position](../flight_modes_mc/position.md) or [Mission](../flight_modes_fw/mission.md) modes (when the vehicle stops moving the static pressure drops, the sensor reports a higher altitude, and the vehicle compensates by descending).
```

Comment:
This docs is about static pressure:

> Air flowing over an enclosed vehicle can cause the _static pressure_ to change within the canopy/hull.
> Depending on the location of holes/leaks in the hull, you can end up with under or overpressure (similar to a wing).

What does overpressure do - i.e. why do I care? Is it something that affects both MC and FW? And if so, which more?

I ask because we should note who is affected and decide whether these links point to FW or MC modes. Currently I set to FW.

Reply:
It affects all vehicles, but it's in general much more visible on MC because a FW moves with a rather constant airspeed. In the end it's airspeed deltas that are noticeable.

Reply:
Cool. I added a comment to note this.